### ISSUE TITLE
HTML5 input types browser validation error not announced (IE11)

### ISSUE THREAD

**Issue by cyberseraphic** [FreedomScientific/VFO-standards-support#14]
### Summary

JAWS 18 does not read the native browser validation error message tooltip in IE11 (have not tested for other versions).

### Expected result

The tooltip should be announced on appearance, and also announced when the input field receives focus and/or associated with the input label.

### Actual result

The error message is completely ignored.

### Example

Example URL:
https://www.webpagefx.com/blog/images/assets/cdn.sixrevisions.com/demos/0345-new_html5_form_input_types/new-html5-form-input-types.html?

Steps to reproduce:
- Open JAWS and IE11
- Navigate to example URL above
- Browse to the example for "email" input type
- Enter an invalid email address and press "Enter"
- Observe the appearance of a tooltip indicating an invalid entry
- Attempt to locate this tooltip by browsing forwards and backwards around the element with JAWS

#### JAWS version and build number
18.0.4350
 
#### Operating System and version
Windows 10 Professional (10.0.16299 Build 16299)

#### Browser and version:
Internet Explorer 11 (11.14.16299.0)

**Comment by patrickhlauke** [comment]
Does this work in any other browser/AT combination? I'll do some checking, but from memory this is a pervasive problem at the moment (down to browsers not exposing those native error messages via the accessibility API)

**Comment by cyberseraphic** [comment]
It works in NVDA on Firefox (latest versions of both) on the same machine, and Talkback and VoiceOver both announce the error message on their respective mobile devices (again, latest versions of both platforms).

So JAWS was the only platform I tested that did not announce the error.

**Comment by stevefaulkner** [comment]
better test case https://thepaciellogroup.github.io/AT-browser-tests/test-files/required-att.html
reproduced with JAWS 18.0.453 and IE11
The error message content is exposed via UIA CurrentHelpText property, so should be conveyed by JAWS

**Comment by stevefaulkner** [comment]
@cyberseraphic filed internal Bug #97582 - HTML5 input types browser validation error not announced (IE11)

**Comment by TPGJRogers** [comment]
Issue only impacted IE11. Closing.
